### ISSUE TITLE
v1 requirements

### ISSUE THREAD

**Issue by LaurentVB** [arhs/iban.js#64]
This issue tracks the requirements for the rewrite of iban.js
- [ ] no polyfill, written as modern js (a compiled version for older browsers could be envisioned)
- [ ] specifications format based on the standard published by Swift (see https://www.swift.com/standards/data-standards/iban)
- [ ] extensible
- [ ] improved documentation

**Comment by MickL** [comment]
I agree with all points, especially having no polyfill which should be added by the surrounding app in which iban is used within. 

Also better imports would be nice: `import { IBAN } from 'iban';` instead of `import * as IBAN from 'iban';`.
